test/e2e/data/cluster-templates/docker-rke2-topology.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       value: ${RKE2_CNI}
     - name: dockerImage
       value: kindest/node:${KUBERNETES_VERSION}
-    version: ${RKE2_VERSION}
+    version: ${KUBERNETES_VERSION}+rke2r1
```

Comment:
we are not using bumped version of RKE2 here (KUBERNETES_VERSION=v1.31.4), is it on purpose?